packages/bitcore-node/src/models/coin.ts

```diff
@@ -67,19 +68,15 @@ class CoinModel extends BaseModel<ICoin> {
 
   async getBalance(params: { query: any }): Promise<{ confirmed: number, unconfirmed: number, balance: number }> {
     let { query } = params;
-    query = Object.assign(query, {
-      spentHeight: { $lt: SpentHeightIndicators.minimum },
-      mintHeight: { $gt: SpentHeightIndicators.conflicting }
-    });
-    const result =  await this.collection
+    const result = await this.collection
       .aggregate<{ _id: string, balance: number }>([
```

Comment:
Type here has changed, should be fixed, which would get rid of the "as" type thing after it